### PR TITLE
clean up readme

### PR DESCRIPTION
**Pull request by gregtampa**
Short Description: 
Made a bunch of wiki entries made readme shorter.
Still needs some work.

### CODE REVIEW

**Comment by douglascamata** [comment]
👍

**Comment by TheSavior** [comment]
:+1:

File: README.md [hunk 1]
```diff
@@ -51,37 +46,30 @@ No PR on master branch to keep things easier.
...
+ * Google Map API key setup
  * Show all objects on map (In Testing)
  * Evolve pokemons (Code in, Need input, In Testing)
-
+ * Incubate eggs
 ## TODO List
 
 - [ ] Standalone Desktop APP
 - [ ] Pokemon transfer filter ?? This already done, right?
 - [ ] Hatch eggs
```
Comment (reddivision) [inline README.md]: we have egg hatching too

**Comment by gregtampa** [comment]
added more features sorry.. need both of you to approve again.

**Comment by solderzzc** [comment]
Need a merge...
👍

**Comment by solderzzc** [comment]
Let me close this one since https://github.com/PokemonGoF/PokemonGo-Bot/pull/1805 in

**Pull request closed without merging**